### DISCUSSION TITLE
allow skipping/ pausing more than just one day

### DISCUSSION THREAD

**Discussion by markus-hsk** [iSoron/uhabits#1300] [Feature Requests]
I know theres a function to allow skipping a single day and i saw a feature request to place this function not globally but per habit #973. My request is an extension to that.

I have a goal to take medicine daily. But there is upcoming a period in which i have to skip that medicine for two weeks because of reasons. Dont going into details. Currently i have to check the daily goal although i did not really took that pill. It's okay for me, because for me the app is only a game or tool or something to make me feel good doing things. It has no relevance for my medical situation. But it would be cool to even be able to set a pause in the tracking without breaking the statistics and setting wrong values. 

I am thinking of it like the automatically placed checks if you have a "x times in a week" habit but now manually set by the user.

**Comment by markus-hsk** [comment]
Uhm, i just discovered there's a function which seems to do exactly what i am asking for. I guess this feature-request can be closed then. Sorry.

**Comment by ke352802081770314** [comment]
Would you share what you found so other readers can also learn about it? I wouldn't know how to achieve that. If the calendar allowed entering future days, then yes, I could already enter skip for the coming two weeks, but it only goes until today (which usually makes sense because we shouldn't enter predictions but only hard facts)

**Reply by markus-hsk** [reply]
There's an config option to skip days. This one has to be active. After that you can long-press a day twice to place a "-" signalizing that on that day the achievment was not done but planned and shall not affect the rating.

**Reply by ke352802081770314** [reply]
You already acklowledged that you are aware of the skip option. I understood your first reply to say that you are now able to enter this in advance for the period that the goal is not relevant. If the solution is just to enter skip for multiple days in sequence, *after each day has already arrived* and not ahead of time, then there's no news for me. 
Many thanks for clarifying.

**Comment by bkoch53** [comment]
Can we maybe extend this ask? I have a similar desire to skip certain habits for extended periods. While the skip is available, if a habit needs to be skipped for longer than a sort time, it can get tedious. Something like a toggle in the specific habit to `pause` the habit via an auto `skip` and pause notifications for that habit.

**Reply by q-wertz** [reply]
Same here. Would be nice for tasks that only need to be done seasonal.
E.g. 
- watering the garden
- taking vitamin D
- ...
